Common/Securities/SecurityTransactionManager.cs

```diff
@@ -365,15 +365,25 @@ public List<Order> GetOpenOrders(Func<Order, bool> filter = null)
         }
 
         /// <summary>
-        /// Gets open orders matching the specified filter. Specifying null will return an enumerable
-        /// of all open orders.
+        /// Gets open orders matching the specified filter. However this method can be confused with the
+        /// override that takes a Symbol as parameter. For this reason it first checks if it can convert
+        /// the parameter into a symbol. If that conversion cannot be aplied it assumes the parameter is
+        /// a Python function object and not a Python representation of a Symbol.
```

Comment:
Nice catch! Can the same happen for some of the other methods here, like `GetOpenOrderTickets`? Would be nice to expand the test calls a bit to cover all possible cases like these too, another example missing to test is `GetOrders()` I think?